packages/javascript/src/__tests__/dispatcher.test.ts

```diff
@@ -0,0 +1,27 @@
+import { Dispatcher } from "../dispatcher"
+import { Queue } from "../queue"
+import { PushApi } from "../api"
+import { Span } from "../span"
+
+jest.mock("../api")
+
+describe("Dispatcher", () => {
+  let dispatcher: Dispatcher
+
+  beforeEach(() => {
+    /* Runs before each test */
```

Comment:
For me personally; this feels like a very verbose comment, a bit in the area of having a method with the name `createUser`, and then have a comment above it: `//Create user`.

I generally prefer to only have comments to document a function, or to explain why a certain trade-off has been made.

It's up to you if you want to change this or not, just wanted to give my 2 cents :-)

Reply:
This was copy and pasted from a template, so my bad for leaving this in. I'll remove this comment, but will make an issue to remove this from the other files. 👍

Reply:
Fixed in f9c2660.